### ISSUE TITLE
Create full-network dump, routable if possible

### ISSUE THREAD

**Issue by mileswwatkins** [orma/openroads-vn-analytics#79]
Output format should be in OSM XML, for easiest integration with Rural Accessibility Map.

**Comment by mileswwatkins** [comment]
Full-network OSM XML dump pushed to phase 2

**Comment by mileswwatkins** [comment]
There should be both a routable and a non-routable dump, per discussion with @sarahantos and @PCDuc this morning. The non-routable dump will be useful for non-core users such as Duc, running ad-hoc analytics. It shouldn't be surfaced, generally speaking, to most users.

**Comment by geohacker** [comment]
Duplicate of https://github.com/orma/openroads-vn-analytics/issues/351
